cmd/root.go

```diff
@@ -61,6 +67,8 @@ var p providers.Provider
 var rm *manager.ResourceManager
 var apiConfig vkubelet.APIConfig
 var podSyncWorkers int
+var listWatcher *cache.ListWatch
+var watchSyncInterval = time.Minute
```

Comment:
Is there the need for such an aggressive interval? I would even argue it should be entirely disabled. See https://groups.google.com/forum/#!topic/kubernetes-sig-api-machinery/PbSCXdLDno0.

Reply:
Mostly just copying what was already there. Thinking about it, I agree we probably don't need to  sync so aggressively.